### ISSUE TITLE
When I add edit text and list view to the current application context, if I click on edit text the keyboard will pop up but edit text remains below the keyboard, it fails to come up with the keyboard and I can't see what I'm typing but if I remove list view then it works fine help fix this pls
this bug currently in 6.4.0 rc01-5 I don't know about other versions though 

### ISSUE THREAD

**Issue by Alu-Kard** [Sketchware-Pro/Sketchware-Pro#818]

**Comment by DereckySany** [comment]
add image or video of the event.

**Comment by JavkhlanK** [comment]
Yeah, please describe your problem better with images and/or screen recordings and not just 1 block of text. Using sentences and more paragraphs will make your explanation easier to read.

**Comment by HAKANKOKCU** [comment]
add your edittext inside scroll(v) and make your activity resizeable.

**Comment by TechnicalStudioDeveloper** [comment]
> Yeah, please describe your problem better with images and/or screen recordings and not just 1 block of text. Using sentences and more paragraphs will make your explanation easier to read.

[Video](https://dl.dropbox.com/s/2mjaoxbg5z33a45/Android%20Code%20Editor.mp4?dl=0)

**Comment by Alu-Kard** [comment]
https://github.com/Sketchware-Pro/Sketchware-Pro/assets/118820844/1981a96f-7bcc-4555-89e3-de0734ecbb4a

This is the Video Guys
Pls help me
